.gitignore

```diff
@@ -3,3 +3,5 @@ dist
 debug.log
 logdump.json
 dist-extra
+script/.DS_Store
+.DS_Store
```

Comment:
instead of this, set up your global gitignore to exclude `.DS_Store

```
$ cat ~/.gitconfig
[core]
  excludesfile = ~/.gitignore
  ```
  
  ```
$  cat ~/.gitignore
.DS_Store
```